com.unity.visualeffectgraph/Editor/Compiler/VFXCodeGenerator.cs

```diff
@@ -751,12 +753,23 @@ private static StringBuilder TryBuildFromShaderGraph(VFXShaderGraphParticleOutpu
             graph.ValidateGraph();
 
             // Check the validity of the shader graph (unsupported keywords or shader property usage).
-            if (!VFXLibrary.currentSRPBinder.IsGraphDataValid(graph))
+            if (VFXLibrary.currentSRPBinder == null || !VFXLibrary.currentSRPBinder.IsGraphDataValid(graph))
```

Comment:
What's the implication of having IsGraphDataValid returning false here? Will it skip the compilation? Make it later? Try to compile anyway with bad state?

Reply:
Also wasnt this path (compilation) not supposed to be hit at all when VFXLibrary.currentSRPBinder is null? (typically when no SRP is installed)

Reply:
This is a known issue we already have in HDRP, a fogbugz has been registered:
https://fogbugz.unity3d.com/f/cases/1364194/